templates/groups/single/admin/email-options.php

```diff
@@ -0,0 +1,93 @@
+<?php if ( groups_is_user_admin( bp_loggedin_user_id(), bp_get_current_group_id() ) || is_super_admin() ) :
```

Comment:
I would move the conditionals back to screen-admin.php.  This logic isn't needed in a template part.

Reply:
I left these conditionals here so that you could decide to show stuff using this template to users who don't meet the conditions in question. But I guess that's an unlikely use case, and I don't much care one way or another, so I'll make the change.